README.md

```diff
@@ -1,8 +1,17 @@
 # hal
-Easily create and manage Kubernetes applications using Dekorate and the Halkyon operator, made with ❤️ by the Snowdrop team.
+
+## Overview
+`hal` is a CLI tool for developers to create and manage Spring Boot applications. Simplify the deployment of applications on OpenShift and Kubernetes using Dekorate and Halkyon Component Operator. Made with ❤️ by the Snowdrop team.
```

Comment:
This is true for the moment as only spring boot project is scaffolded but not at all regarding to what `hal` proposes globally. I propose to rephrase to `Hal is a CLI tool for developers to simplify the deployment of applications such as Spring Boot on OpenShift and Kubernetes using Dekorate and Halkyon Component Operator. Made with ❤️ by the Snowdrop team.`